execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -281,6 +319,7 @@ impl LmdbWasmTestBuilder {
             handle_payment_contract_hash: None,
             standard_payment_hash: None,
             auction_contract_hash: None,
+            scratch_engine_state: None,
```

Comment:
Why do we not create it immediately in the constructor?

Reply:
I've got my partial answer - the `get_scratch_global_state` is not defined on `InMemGlobalState`.